src/network/msgUtils.go

```diff
@@ -83,12 +101,18 @@ func FormatSyncMsg(
 	syncTarget int,
 	recipient int,
 	author int,
+	loopCounter ...int,
```

Comment:
I realize we don't need the ability to set the counter to a specific value in these formatting functions as we only use them when creating a new message.